src/profile/external-identities/OAuth2Providers.ts

```diff
@@ -98,6 +104,7 @@ export const oauth2Provider = (providerKey: OAuth2ProviderKey): OAuth2Provider =
         supportsAccessToken: true,
         supportsIdToken: false,
         isFence: true,
+        toolTip: tooltipMessages.kids_first,
```

Comment:
My first reaction was to simplify by inlining the tooltip content right in this file instead of making a separate file. I realize the tooltip content is somewhat verbose. I'll leave it up to you; I could go either way on it.

Reply:
Fixed: https://github.com/DataBiosphere/terra-ui/pull/5259/commits/f6c3489eb5088c1bfff3e0a57c7870d049383305